pkg/ddevapp/config.go

```diff
@@ -90,6 +106,16 @@ func (c *Config) Write() error {
 		return err
 	}
 
+	cfgbytes = append(cfgbytes, []byte(HookTemplate)...)
+	switch c.AppType {
+	case "drupal8":
+		cfgbytes = append(cfgbytes, []byte(Drupal8Hooks)...)
+	case "drupal7":
+		cfgbytes = append(cfgbytes, []byte(Drupal7Hooks)...)
+	case "wordpress":
+		cfgbytes = append(cfgbytes, []byte(WordPressHooks)...)
+	}
+
```

Comment:
I feel like doing nice things for applications we know about is OK, but we may want to think about the following:

1. Perhaps find a way to consolidate this app specific logic so it's not so scattered everywhere. Currently it sort of randomly resides all over the place.
2. Make sure we always provide a no-op or "other" that handles applications we don't know about in a sane way.